src/app/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.scss

```diff
@@ -1,22 +1,22 @@
 @import '../../../../assets/stylesheets/base';
-codebases-item-workspaces { display: flex; }
+
+codebases-item-workspaces {
+  display: flex;
+}
+
 .busy-indicator {
-  position: relative;
   display: flex;
   height: auto;
-  width: 25px;
   margin-left: 10px;
+  position: relative;
```

Comment:
It's still not clear what style guide you're referring to -- Bootstrap 4? Can you please provide a pointer?